entity-framework/core/querying/related-data/eager.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Eager Loading of Related Data - EF Core
+description: Eager loading of related data with Entity Framework Core
+author: roji
+ms.date: 9/8/2020
+uid: core/querying/related-data/eager
+---
+# Eager Loading of Related Data
+
+## Eager loading
+
+You can use the `Include` method to specify related data to be included in query results. In the following example, the blogs that are returned in the results will have their `Posts` property populated with the related posts.
+
+[!code-csharp[Main](../../../../samples/core/Querying/RelatedData/Sample.cs#SingleInclude)]
+
+> [!TIP]
+> Entity Framework Core will automatically fix-up navigation properties to any other entities that were previously loaded into the context instance. So even if you don't explicitly include the data for a navigation property, the property may still be populated if some or all of the related entities were previously loaded.
+
+You can include related data from multiple relationships in a single query.
+
+[!code-csharp[Main](../../../../samples/core/Querying/RelatedData/Sample.cs#MultipleIncludes)]
+
+## Including multiple levels
+
+You can drill down through relationships to include multiple levels of related data using the `ThenInclude` method. The following example loads all blogs, their related posts, and the author of each post.
+
+[!code-csharp[Main](../../../../samples/core/Querying/RelatedData/Sample.cs#SingleThenInclude)]
+
+You can chain multiple calls to `ThenInclude` to continue including further levels of related data.
+
+[!code-csharp[Main](../../../../samples/core/Querying/RelatedData/Sample.cs#MultipleThenIncludes)]
+
+You can combine all of the calls to include related data from multiple levels and multiple roots in the same query.
+
+[!code-csharp[Main](../../../../samples/core/Querying/RelatedData/Sample.cs#IncludeTree)]
+
+You may want to include multiple related entities for one of the entities that is being included. For example, when querying `Blogs`, you include `Posts` and then want to include both the `Author` and `Tags` of the `Posts`. To include both, you need to specify each include path starting at the root. For example, `Blog -> Posts -> Author` and `Blog -> Posts -> Tags`. It doesn't mean you'll get redundant joins; in most cases, EF will combine the joins when generating SQL.
+
+[!code-csharp[Main](../../../../samples/core/Querying/RelatedData/Sample.cs#MultipleLeafIncludes)]
+
+## Single and split queries
```

Comment:
Need to update the fwlink to point here https://go.microsoft.com/fwlink/?linkid=2134277

Reply:
/cc @JeremyLikness 

Do we have a list somewhere tracking needed changes or something?

Reply:
I am doing exception message review. 😆

Reply:
@roji I don't have a master list of fwlinks to update, only what people reach out to me to ask.

Reply:
@roji @JeremyLikness - Can we get this link target updated? It has been pushed to live. And exception message leaves in source code already.

Reply:
This is just about changing the fwlink right?

Reply:
Yes.

Reply:
@roji @smitpatel what is the URL it needs to point to?

Reply:
https://docs.microsoft.com/ef/core/querying/related-data/eager#single-and-split-queries

Reply:
Thanks. This is done.